### PR TITLE
Upgrade Python ecoCode plugin to 1.4.2

### PR DESCRIPTION
**Pull request by dedece35**
Hi,

ecoCode Python plugin 1.4.2 has been released.

Changes from last SonarQube marketplace integration (1.3.0), please check following changelogs :

from 1.3.0 to 1.4.0 : https://github.com/green-code-initiative/ecoCode-python/releases/tag/1.4.0
from 1.4.0 to 1.4.1 : https://github.com/green-code-initiative/ecoCode-python/releases/tag/1.4.1
from 1.4.1 to 1.4.2 : https://github.com/green-code-initiative/ecoCode-python/releases/tag/1.4.2

The corresponding SonarCloud project: https://sonarcloud.io/project/overview?id=green-code-initiative_ecoCode-python

Please update the market place.

Thank you

Regards

### CODE REVIEW

File: ecocodepython.properties [hunk 1]
```diff
@@ -14,7 +14,13 @@ defaults.mavenArtifactId=ecocode-python-plugin
 1.2.1.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.2.1
 
 1.3.0.description=Updates rules tags�; adds rules (EC10, EC66, EC203, EC404)�; upgrade librairies to SonarQube 10.0.0�; clean-up plugins and dependencies
-1.3.0.sqVersions=[9.9,LATEST]
+1.3.0.sqVersions=[9.9,10.1]
```
Comment (ganncamp) [inline ecocodepython.properties:17]: This range needs to end with the current version so that your current users on SQ 10.3 will be prompted to upgrade.

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: ecocodepython.properties (hunk 1 above)
Comment (dedece35) [reply ecocodepython.properties:17]: ok. thank you. I'm going to do this

Comment (dedece35) [reply ecocodepython.properties]: done

File: ecocodepython.properties [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+homepageUrl=https://github.com/green-code-initiative/ecoCode-python
+publicVersions=1.4.2
+archivedVersions=1.2.1,1.3.0
 
 defaults.mavenGroupId=io.ecocode
 defaults.mavenArtifactId=ecocode-python-plugin
 
 1.2.1.description=Initial Marketplace publication of plugin for SonarQube 9.9.+ LTS
-1.2.1.sqVersions=[9.9,10.1]
+1.2.1.sqVersions=[9.9,LATEST]
```
Comment (ganncamp) [inline ecocodepython.properties:11]: Please see my comments on the Java plugin. And I apologize for the confusion.

File: ecocodepython.properties [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
 1.2.1.date=2023-04-14
 1.2.1.downloadUrl=https://github.com/green-code-initiative/ecoCode/releases/download/1.2.1/ecocode-python-plugin-1.2.1.jar
 1.2.1.changelogUrl=https://github.com/green-code-initiative/ecoCode/releases/tag/1.2.1
 
-1.3.0.description=Updates rules tags�; adds rules (EC10, EC66, EC203, EC404)�; upgrade librairies to SonarQube 10.0.0�; clean-up plugins and dependencies
+1.3.0.description=Updates rules tags; adds rules (EC10, EC66, EC203, EC404); upgrade librairies to SonarQube 10.0.0; clean-up plugins and dependencies
 1.3.0.sqVersions=[9.9,LATEST]
```
Comment (ganncamp) [inline ecocodepython.properties:17]: End this range with `10.3`

**Review by ganncamp — CHANGES_REQUESTED** [review]

**Review by ganncamp — APPROVED** [review]

**Pull request merged**